test/regression/internal/reset-password.spec.js

```diff
@@ -24,51 +22,73 @@ describe('internal user resetting their password:', function () {
     expect($('input#email')).toBeVisible();
   });
 
-  it('shows a validation message if the email field is empty', () => {
-    $('input#email').setValue('');
-    getButton('Continue').click();
+  it('shows a validation message if the email field is empty', async () => {
+    const email = await $('#email');
+    const continueButton = await getButton('Continue');
+    const validationMessage = await getValidationSummaryMessage();
 
-    expect(getValidationSummaryMessage()).toHaveText('Enter an email address');
+    email.setValue('');
+    continueButton.click();
+    expect(validationMessage).toHaveText('Enter an email address in the right format');
```

Comment:
Same as with external password reset. I'm not sure that this validation message is correct

Reply:
✅